src/reducers/scenes/WalletsReducer.js

```diff
@@ -169,7 +169,7 @@ const walletLoadingProgress = (state = {}, action: Action): $PropertyType<Wallet
       if (!action.data) throw new Error('Invalid action')
       const activeWalletIdList = action.data.activeWalletIds
       const activeWalletIdProgress = {}
-      activeWalletIdList.map(item => {
+      activeWalletIdList.forEach(item => {
```

Comment:
How about just using reduce:
```
return activeWalletIdList.reduce((r, item) => ({...r, [item]: 0 }), {})
```

Reply:
I didn't want to do a big refactor on any of the code, just make ESLint happy. Also, the replacement you suggest is O(n^2), not O(n) like the current version, so it's not clearly superior.